bmp.h

```diff
@@ -1,41 +1,41 @@
-#include <cstdint>
+
+#include <cstdint.h>
 
 typedef uint8_t BYTE;
 typedef uint32_t DWORD;
 typedef int32_t LONG;
 typedef uint16_t WORD;
```

Comment:
```suggestion
typedef uint8_t   BYTE;
typedef uint32_t DWORD;
typedef int32_t   LONG;
typedef uint16_t WORD;
```
Follow a coding style standard.